packages/calypso-e2e/src/hooks/browser.ts

```diff
@@ -8,14 +8,25 @@ import { Context } from 'mocha';
  */
 import { start, close } from '../browser-manager';
 
+import { Page } from 'playwright';
+
+declare global {
```

Comment:
Instead of the `this` context used by mocha, jest uses the `global` context.

TypeScript introduces a problem and this approach patches the global such that page objects can be assigned. See [this StackOverflow](https://stackoverflow.com/questions/40743131/how-to-prevent-property-does-not-exist-on-type-global-with-jsdom-and-t).

Additionally, it seems the maintainers of `jest-playwright-preset` also use the global scope to manage page, browsercontext and other attributes, so I am not alone in this.